lightning/src/ln/msgs.rs

```diff
@@ -41,7 +41,7 @@ use util::ser::{Readable, Writeable, Writer, FixedLengthReader, HighZeroBytesDro
 use ln::channelmanager::{PaymentPreimage, PaymentHash, PaymentSecret};
 
 /// 21 million * 10^8 * 1000
-pub(crate) const MAX_VALUE_MSAT: u64 = 21_000_000_0000_0000_000;
```

Comment:
This is very specifically written this way because the number is 21 million * 10^8, and thus grouping the 8 into 2*4 is more readable. Same with the other SATS numbers.